gradle.properties

```diff
@@ -9,7 +9,7 @@ pluginVersion = 1.9.1
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
 pluginSinceBuild = 203
-pluginUntilBuild = 213.*
+pluginUntilBuild = 222.*
```

Comment:
```suggestion
```

Remove the until build 
```suggestion
```

Remove UntilBuild to run continully version

Reply:
Sorry the pluginUntilBuild option is mandatory.

Reply:
Yeah I found out that the hard way too.. Thanks for leaving a comment